### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by romanlex** [helljs/eslint-import-resolver-x#1] [General]
You can post you feedback after replace typescript resolver on `x` resolver and show everyone how much faster linting is performed in CI

**Comment by JounQin** [comment]
Hi, maintainer of `eslint-import-resolver-typescript`here.

I'm not sure to understand why not contributing to original `eslint-import-resolver-typescript` instead?

By the way, we just released a fresh new [`v4.0.0`](https://github.com/import-js/eslint-import-resolver-typescript/releases/tag/v4.0.0) with great performance improvements.

Hope we can collaborate to improve the resolver(s).
